proxygen_cli/test/command_credentials_test.py

```diff
@@ -63,21 +63,28 @@ def test_get_invalid_setting(patch_config):
 
         assert "'invalid' is not one of 'base_url'" in result.output.strip()
 
-@patch.object(Credentials, '_validate_private_key_path')
+
 def test_private_key_id_missing(patch_config):
     """
     Ensure that if the private_key_path is set, then we validate key_id.
-    We mock out the file path validation using an extra patch as we're already mocking part of Credentials 
-    with the patch_config mark.
+    We mock out the file path validation using an extra patch as we're already
+    mocking part of Credentials with the patch_config mark.
     """
     mock_credentials = get_test_credentials()
 
-    with patch_config(credentials=mock_credentials):
+    with patch_config(credentials=mock_credentials),\
```

Comment:
Unrelated change ,but this test was failing due to to mocked property `_validate_private_key_path` not being serialisable. I've rewritten the mocking slightly to provide a real return value, which can be serialised.